dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ORCIDHelper.java

```diff
@@ -0,0 +1,127 @@
+package io.dockstore.webservice.helpers;
+
+import java.io.IOException;
+import java.io.StringWriter;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.Optional;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+import javax.xml.datatype.DatatypeConfigurationException;
+import javax.xml.datatype.DatatypeFactory;
+import javax.xml.datatype.XMLGregorianCalendar;
+
+import io.dockstore.webservice.core.Entry;
+import io.dockstore.webservice.core.Version;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.orcid.jaxb.model.common.Relationship;
+import org.orcid.jaxb.model.common.WorkType;
+import org.orcid.jaxb.model.v3.release.common.CreatedDate;
+import org.orcid.jaxb.model.v3.release.common.LastModifiedDate;
+import org.orcid.jaxb.model.v3.release.common.Subtitle;
+import org.orcid.jaxb.model.v3.release.common.Title;
+import org.orcid.jaxb.model.v3.release.common.Url;
+import org.orcid.jaxb.model.v3.release.record.ExternalID;
+import org.orcid.jaxb.model.v3.release.record.ExternalIDs;
+import org.orcid.jaxb.model.v3.release.record.Work;
+import org.orcid.jaxb.model.v3.release.record.WorkTitle;
+
+public final class ORCIDHelper {
+    // TODO: Change to non-sandbox for Dockstore use while using sandbox for test
+    private static final String ORCID_SANDBOX_BASE_URL = "https://api.sandbox.orcid.org/v3.0/";
```

Comment:
Should probably ultimately be configurable, e.g., we might want to keep it as sandbox in staging/dev.

Reply:
Will leave it to the other issue that deals more with the web service hook up